tests/pl/light_HassTurnOn.yaml

```diff
@@ -3,6 +3,7 @@ tests:
   - sentences:
       - "Zapal światło w kuchni"
       - "Zapal wszystkie światła w kuchni"
+      - "Zaświeć całe światła w kuchni"
```

Comment:
The correct form should be "Zaświeć całe światło w kuchni"

Reply:
Hmm, the whole `Całe` was added in another PR and I see the exact same issue as you do, I think this is in fact grammatically incorrect, converting `światła` to `światło` for this specific form will be somewhat problematic because it applies also for other different forms. I'll give it a go later today and try to find suitable solution, thanks for the feedback.